src/quo2/README.md

```diff
@@ -69,6 +69,38 @@ In the image above we can see the properties are `Type`, `State`, `Size`,
   ...)
 ```
 
+### Handling Sizes
+For sizes there is a slightly special case. In the designs, sizes are referred to as integers. 
```

Comment:
When reading the guideline in isolation, the first sentence seems unnecessary. We can start the paragraph with `In the designs, sizes ...`.

Suggestion, with line breaks at 80:

```markdown
In the designs, sizes are referred to as integers. To avoid having the codebase
littered with magic numbers we instead have a keyword convention to use in
components to map these keywords with their sizes.
```